docs/can-canjs/can-api.md

```diff
@@ -91,6 +91,12 @@ customElements.define("my-counter", Counter);
 
 Also, the [api#ElementBindings Element Bindings] section shows how to pass data between components.
 
+ [customElements.define](https://developer.mozilla.org/en-US/docs/Web/API/CustomElementRegistry/define) allows developers to define a new custom HTML tags by taking a custom element tag name and a JavaScript class:
+ 
+ ```js
+ customElements.define("my-counter", Counter);
+ ```
+
 
 ## Observables
```

Comment:
This section is still very large. I think we should just show a simple example like

```js
class Todo extends ObservableObject {
  static props = {
    name: string,
    complete: false
  };

    // `toggleComplete` is a method
    toggleComplete() {
        this.complete = !this.complete;
    }
}

// Create a todo instance:
const todo = new Todo({ name: "Learn Observables" });
```

and then add expandable sections for all of the other types for properties.

Reply:
Done!